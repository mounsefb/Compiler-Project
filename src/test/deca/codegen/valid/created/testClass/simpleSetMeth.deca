class Point{
    int x, y;
    void set(int x, int y) {
        this.x = x;
        this.y = y;
    }
}
{
Point p = new Point();
p.set(3, 4);
println("p.x = ", p.x, "    p.y=", p.y);
}