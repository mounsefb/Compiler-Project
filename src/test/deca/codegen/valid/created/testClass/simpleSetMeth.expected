p.x = 3    p.y=4